cmd/envtool/testdata/error_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testdata
+
+import "testing"
+
+func TestError1(t *testing.T) {
+	t.Error("Error 1")
```

Comment:
Let's add `t.Log` lines everywhere to see what output will be hidden and what not